docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -725,14 +726,16 @@ In this case, the table names are encoded in the job definition. If, instead, yo
 ```python file=/concepts/io_management/metadata.py startafter=io_manager_start_marker endbefore=io_manager_end_marker
 class MyIOManager(ConfigurableIOManager):
     def handle_output(self, context: OutputContext, obj):
-        table_name = context.metadata["table"]
-        schema = context.metadata["schema"]
-        write_dataframe_to_table(name=table_name, schema=schema, dataframe=obj)
+        if context.metadata:
```

Comment:
i know this is for making typing happy and we should almost never reach the `else` state, but bc it's a critical path for io manager, should we handle the `else` state to do some error handling?